### PR TITLE
10 new color schemes

### PR DESCRIPTION
**Pull request by schnensch0**
added 10 new color schemes to ziro, updated blue-dark and blue-light and made some small fixes

### CODE REVIEW

File: THEMES.md [hunk 1]
```diff
@@ -78,10 +78,43 @@ https://user-images.githubusercontent.com/19476925/119483404-84223100-bd87-11eb-
 
 ## Ziro
 
-#### Blue Dark
+### Blue Dark
 
 ![Album](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/album-blue-dark.png)
 
-#### Blue Light
+### Blue Light
```
Comment (morpheusthewhite) [inline THEMES.md:85]: Use `####` instead, as specified in the README

File: THEMES.md [hunk 2]
```diff
@@ -78,10 +78,43 @@ https://user-images.githubusercontent.com/19476925/119483404-84223100-bd87-11eb-
...
+
+### Purple Light
+![Playlist](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/playlist-purple-light.png)
+
+### Red Dark
+![Profile](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/profile-red-dark.png)
+
+### Red Light
+
+![Queue](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/queue-red-light.png)
```
Comment (morpheusthewhite) [inline THEMES.md:120]: These color schemes somehow follow the same pattern. Can you code an HTML table to make them more compact? Somethink like https://github.com/morpheusthewhite/spicetify-themes/tree/v2/Onepunch#dark

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

File: THEMES.md (hunk 2 above)
Comment (schnensch0) [reply THEMES.md:120]: would it be possible to do something like this:
#### Blue
| Album | Artist |
| :-: | :-: |
| Dark | Light |

#### Gray
| Cards | Search |
| :-: | :-: |
| Dark | Light |

like group them by color?

Comment (morpheusthewhite) [reply THEMES.md:120]: I think it's fine

**Review by morpheusthewhite — APPROVED** [review]

**Pull request merged**